plugin/src/main/resources/com/google/tsunami/plugin/payload/payload_definitions.yaml

```diff
@@ -92,3 +92,12 @@ payloads:
     validation_regex: (?s).*TSUNAMI_PAYLOAD_START$TSUNAMI_PAYLOAD_TOKEN_RANDOMTSUNAMI_PAYLOAD_END.*
     vulnerabilityType:
       - REFLECTIVE_RCE
+  - name: jsp_print
+    interpretation_environment: JSP
+    execution_environment: EXEC_INTERPRETATION_ENVIRONMENT
+    uses_callback_server: false
+    payload_string: <% out.print(String.format("%s%s%s","TSUNAMI_PAYLOAD_START", "$TSUNAMI_PAYLOAD_TOKEN_RANDOM", "TSUNAMI_PAYLOAD_END")); %>
+    validation_type: VALIDATION_REGEX
+    validation_regex: (?s).*TSUNAMI_PAYLOAD_START$TSUNAMI_PAYLOAD_TOKEN_RANDOMTSUNAMI_PAYLOAD_END.*
+    vulnerability_type:
+      - REFLECTIVE_RCE
```

Comment:
nit: add a Terminating Newline at the end of the file.

Reply:
Once this PR is merged, could you rebase https://github.com/google/tsunami-security-scanner-plugins/pull/566?

Reply:
> nit: add a Terminating Newline at the end of the file.

Thanks, Newline added.

Reply:
> Once this PR is merged, could you rebase [google/tsunami-security-scanner-plugins#566](https://github.com/google/tsunami-security-scanner-plugins/pull/566)?

The detector code uses the latest available release of the scanner (`tsunamiVersion = 'latest.release'`), so I believe once this PR is merged, the detector code will properly run, as it will download the latest version with newly added JSP payload config.